helm/kanister-operator/values.yaml

```diff
@@ -89,3 +89,20 @@ tolerations: []
 #
 # node labels for pod assignment. Evaluated as template
 nodeSelector: {}
+
+# Pod-level security context settings
+podSecurityContext:
+#  runAsUser: 1000
+#  fsGroup: 2000
+#  runAsNonRoot: true
+
+# Container-level security context settings
+containerSecurityContext:
+#  capabilities:
+#    drop:
+#    - ALL
+#  readOnlyRootFilesystem: true
+#  runAsNonRoot: true
+# AllowPrivilegeEscalation should be disabled for security
+#  allowPrivilegeEscalation: false
```

Comment:
I think these fields should go under the controller field that we have above. Since we are configuring the the security context of the controller pod. what do you think?

Reply:
I see this is resolved as well, have we moved these fields under the controller field?